SR2025/2025-01-22-competition-announcement.md

```diff
@@ -0,0 +1,20 @@
+---
+to: SR2025 teams
+subject: Student Robotics 2025 Competition weekend confirmed as Saturday 12th to Sunday 13th April
+---
+
+Hi,
+
+We are happy to confirm that [Student Robotics 2025](https://studentrobotics.org/events/sr2025/competition/) will be taking place on **Saturday 12th April to Sunday 13th April** at the University of Southampton Students' Union! We apologise for the delay. The doors will open at **9am** on both days, with the first league matches starting at **11am** on Saturday. Both Saturday and Sunday will have league matches with no teams being eliminated before the Double Elimination Knockout on Sunday afternoon.
+
+Remember the second [challenge](https://studentrobotics.org/docs/resources/2025/challenges.html) deadline is this **Saturday 25th January** at **6pm**.
+
+Next up, the virtual league will be taking place in a couple of weeks. The deadline to submit code is **Thursday 6th February** at **6pm** and the livestream will start at 1pm on Saturday 8th February.
+
+To prepare for  the competition, we have two more tech days coming up in a few weeks:
+* [**15th February 2025** - at Red River Software in Horsham](https://studentrobotics.org/events/sr2025/horsham-tech-day-february)
+* [**8th March 2025** - at Raspberry Pi Foundation in Cambridge](https://studentrobotics.org/events/sr2025/cambridge-tech-day-march)

```

Comment:
Not sure how to suggest wording it, but if we send this as-is before the weekend I expect emails from teams who are expecting to come to the Tech Day this weekend confused/worried that it's being cancelled. (I assume that it's not being cancelled?)

Reply:
https://github.com/srobo/team-emails/pull/206/commits/7b7a491c852cb5bf5171a2194b145723f4d1d485